src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/PrefetchGetRecordsCache.java

```diff
@@ -50,6 +50,17 @@
 
     private boolean started = false;
 
+    /**
+     * Constructor
```

Comment:
Please provide more detail about the class here, and what it's doing.

Reply:
That description is given at the class level. Didn't want to duplicate the same comments. Will add a few lines to describe.

https://github.com/awslabs/amazon-kinesis-client/blob/prefetch/src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/PrefetchGetRecordsCache.java#L29-L36